# x, y: 抓取点的像素坐标 (x, y)
# z: 抓取点的深度距离，单位 米
# grasp_depth: 抓取深度,实际抓取点距离物体表面的深度
# angle: 抓取弧度角 [0, 2π]
# width: 抓取宽度,单位米

uint16 x
uint16 y
float64 z
float64 grasp_depth
float64 angle
float64 width
